### ISSUE TITLE
Lesson Two Practive One does not match text.

### ISSUE THREAD

**Issue by zircher** [GDQuest/learn-gdscript#66]
(https://gdquest.github.io/learn-gdscript/#course/lesson-2-your-first-error/practice-85733.tres)
In this practice you want the use fix the second like with a tab.  But, there is a tab present.  The problem is the '#' character needs to be removed.

So, either change the text to match the actual problem or change the practice by removing the tab and # symbol on the second line.

**Comment by Xananax** [comment]
Do you mean the error text?
![image](https://user-images.githubusercontent.com/836082/147756049-47cc46d2-e8af-4396-b462-21124f733d1e.png)

**Comment by zircher** [comment]
> Do you mean the error text? ![image](https://user-images.githubusercontent.com/836082/147756049-47cc46d2-e8af-4396-b462-21124f733d1e.png)

Okay, this is bizarre.  I went back to the practice and looked at the text and it is correct (remove the #), but when it did it the first time it was talking about adding a tab to fix the problem.  That's... odd.  I guess this is a moot point if the issue has been fixed.  Is there any randomness to the practice that could cause it to sometimes have a different practice problem?
